### ISSUE TITLE
Printing into PDF is failed.

### ISSUE THREAD

**Issue by tarokawa516** [WolframResearch/Chatbook#316]
On Chatbook, I want to print out my session. I use print menu, but I see only error message on printed PDF file... no expected contents seen. 

[printed_file_not_excepted_content_but_error_on_print.pdf](https://github.com/WolframResearch/Chatbook/files/12301301/printed_file_not_excepted_content_but_error_on_print.pdf)

<details>
<summary>Debug Data</summary>

| Property | Value |
| --- | --- |
| Name | ``"Wolfram/Chatbook"`` |
| Version | ``"1.1.1"`` |
| EvaluationEnvironment | ``"Session"`` |
| FrontEndVersion | ``"Desktop: 13.3 for Microsoft Windows (64-bit) (2023å¹´6æ3æ¥)"`` |
| KernelVersion | ``"13.3.0 for Microsoft Windows (64-bit) (June 3, 2023)"`` |
| SystemID | ``"Windows-x86-64"`` |
| Notebooks | ``True`` |
| DynamicEvaluation | ``False`` |
| SynchronousEvaluation | ``False`` |
| TaskEvaluation | ``False`` |

## Settings

| Property | Value |
| --- | --- |
| StyleSheetVersion | ``"1.1.1.3899380025"`` |

## Stack Data
```
throwFailure
trimURL
```

## Failure Data

```
<|
  "Evaluation" :> Wolfram`Chatbook`Actions`AIAutoAssist[cell]
  "Arguments" -> {}
|>
```

</details>

**Comment by rhennigan** [comment]
@tarokawa516 could you share the notebook that you are unable to print? I haven't been able to reproduce this issue when attempting to print chat notebooks to PDF.

[test.pdf](https://github.com/WolframResearch/Chatbook/files/12305622/test.pdf)

**Comment by tarokawa516** [comment]
Sure! here it is. 
[testsample.zip](https://github.com/WolframResearch/Chatbook/files/12307404/testsample.zip)
Since .nb file was rejected, so I put into zip archive.

**Comment by rhennigan** [comment]
I was also unable to print your notebook to PDF, but in my case, it produced an empty file: 
[printed.pdf](https://github.com/WolframResearch/Chatbook/files/12322636/printed.pdf)

It did work when I used File > Save As > PDF though: 
[saved.pdf](https://github.com/WolframResearch/Chatbook/files/12322634/saved.pdf)

It seems like it has something to do with non-ASCII characters appearing in BoxData cells. I ran into the same issue when trying to print a normal notebook that had just one input cell containing the string "日本語の文字".

As far as I can tell, this isn't an issue specific to chat notebooks, so I'm closing this ticket. However, I'll report this internally as a PDF printing issue.
